test/dlc_tests/conftest.py

```diff
@@ -867,6 +868,25 @@ def skip_inductor_test(request):
             )
 
 
+@pytest.fixture(autouse=True)
+def skip_dgl_test(request):
+    if "training" in request.fixturenames:
+        img_uri = request.getfixturevalue("training")
+    elif "pytorch_training" in request.fixturenames:
+        img_uri = request.getfixturevalue("pytorch_training")
+    else:
+        return
+    _, image_framework_version = get_framework_and_version_from_tag(img_uri)
+    image_cuda_version = get_cuda_version_from_tag(img_uri)
+    if request.node.get_closest_marker("skip_dgl_test"):
+        if Version(image_framework_version) in SpecifierSet(">=2.0") and Version(
+            image_cuda_version.strip("cu")
+        ) >= Version("121"):
+            pytest.skip(
+                f"DGL doesn't support cuda12.x for now, so skipping this container with tag {fw_ver}"
```

Comment:
nit: 12.1 instead of 12.x ?

Reply:
DGL actually doesnt support any cuda12.x, so added it. Also I have added `>= Version("121"):` so any cuda 12.x.